instr.ml

```diff
@@ -137,6 +137,7 @@ type program = {
   main : afunction;
   functions : afunction list;
 }
+type analysis_input = ModedVarSet.t * instructions
```

Comment:
You should either have a comment explaining what the first component is (a set of formal parameters), or make this a record: `{ formals : ModedVarSet.t; instrs : instructions }`.